libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -684,6 +743,126 @@ bool AC_Fence::sys_status_failed() const
     return false;
 }
 
+/*
+  get altitude in desired type for a specified location
+ */
+bool AC_Fence::convert_alt_type(const Location &loc, Location::AltFrame desired_type, float alt, float &retalt) const
```

Comment:
```suggestion
  get altitude in desired frame for a specified location
 */
bool AC_Fence::convert_alt_frame(const Location &loc, Location::AltFrame desired_frame, float alt, float &retalt) const
```


similarly elsewhere.  We really don't want to use "type" when we mean "frame".

Reply:
so to be clear, we are naming the parameter FENCE_ALT_TYPE and the corresponding variable should also be named _alt_type for consistency but every where we use "frame".  right @peterbarker?